locale/fi/LC_MESSAGES/django.po

```diff
@@ -17,391 +17,511 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
 
-#: resources/admin.py:12 resources/admin.py:15
-msgid "RESPA Resource booking system"
-msgstr "RESPA Resurssien varausjärjestelmä"
+#: resources/admin/period_inline.py:38
+msgid "HH:mm"
+msgstr ""
```

Comment:
hmm, it is shown to (admin) users as a placeholder, why it should not be translated?